### ISSUE TITLE
0.6.3 error

### ISSUE THREAD

**Issue by fanproductions** [ShadowMario/FNF-PsychEngine#12843]
### Describe your bug here.

help me with this. 
![image](https://github.com/ShadowMario/FNF-PsychEngine/assets/115906994/289e7dd9-3586-492c-92a4-b767ef0d0195)

### Command Prompt/Terminal logs (if existing)

```bash
D:\Nova pasta\FNF-PsychEngine-0.6.3>lime test windows
source/Main.hx:85: characters 61-65 : Float should be Int
source/Main.hx:85: characters 61-65 : ... For optional function argument 'updateFramerate'
source/Main.hx:85: characters 61-65 : ... For function argument 'child'
source/PlayState.hx:75: characters 8-22 : Type not found : vlc.MP4Handler
source/PlayState.hx:1620: characters 13-23 : Type not found : MP4Handler
```

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

No.

**Comment by fanproductions** [comment]
help soon as possible, i need quick help. cuz yes

**Comment by CaptainBaldi** [comment]
dont report bugs for older versions + research these errors have been mentioned and fixed a million times

**Comment by ShadowMario** [comment]
These errors have been fixed on 0.7+, so don't report it
